Sunrise/Helpers/AppRouting.swift

```diff
@@ -71,4 +71,16 @@ class AppRouting {
         homeNavigationController.popToRootViewControllerAnimated(true)
     }
 
+    /**
+        Switches to the account tab, and presents reservation overview view controller.
+    */
+    static func showReservationWithId(reservationId: String) {
+        guard let tabBarController = tabBarController, ordersNavigationController = tabBarController.viewControllers?[2] as? UINavigationController,
+                ordersViewController = ordersNavigationController.viewControllers.first as? OrdersViewController else { return }
+
+        tabBarController.selectedIndex = 2
```

Comment:
Good point, we will do that shortly 👍 

It will be useful when adding or removing tabs, or even for changing their order in the tab controller.